geti_sdk/demos/predict_video.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import logging
+import os
+import shutil
+import tempfile
+import time
+from typing import List, Optional, Union
+
+import cv2
+import ffmpeg
+from tqdm.auto import tqdm
+from tqdm.contrib.logging import logging_redirect_tqdm
+
+from geti_sdk.data_models import Prediction
+from geti_sdk.deployment import Deployment
+from geti_sdk.utils import show_image_with_annotation_scene
+
+
+def predict_video_on_local(
```

Comment:
I think this function name is a bit ambiguous. I'd suggest either `predict_video_locally`,  `predict_video_on_local_machine` or `predict_video_from_deployment` what do you think?

Reply:
`predict_video_from_deployment` would be good. I'll change the function name.